src/tape/ensemble.py

```diff
@@ -1247,9 +1247,10 @@ def convert_flux_to_mag(self, flux_col, zero_point, err_col=None, zp_form="mag",
         ----------
         flux_col: 'str'
             The name of the ensemble flux column to convert into magnitudes.
-        zero_point: 'str'
+        zero_point: 'str' or 'float'
             The name of the ensemble column containing the zero point
-            information for column transformation.
+            information for column transformation. Alternatively, a float zero
```

Comment:
It may be may issue, but `float zero point` sounds a bit weird. Could we say something like
"The name of the ensemble column containing the zero point values. Alternatively, a single float number to apply for all fluxes."